core/src/main/resources/swagger_spec.json

```diff
@@ -1443,6 +1443,22 @@
           "required" : false,
           "type" : "string",
           "default" : "created"
+        }, {
+          "name" : "topic",
+          "in" : "query",
+          "required" : false,
+          "type" : "string"
+        }, {
+          "name" : "environment",
```

Comment:
In `/getTopics` the environment query parameter is called `env`. I also prefer the more explicit name, but I think we could use `env` here as well for the sake of consistency .